src/web/experiments/tests/sign-in-gate-copy-opt.ts

```diff
@@ -0,0 +1,50 @@
+import { ABTest } from '@guardian/ab-core';
+
+export const signInGateCopyOpt: ABTest = {
+	id: 'SignInGateCopyOpt',
+	start: '2021-02-24',
+	expiry: '2021-12-01',
```

Comment:
the expiry could be set at a couple months away, rather than 9 😄

Reply:
I assumed these would be updated once we knew when the test would run

Reply:
We usually just keep a long date, and manually turn it off/remove the test once we're done. Which is why I usually set a long date.

Reply:
> We usually just keep a long date, and manually turn it off/remove the test once we're done. Which is why I usually set a long date.

I think the expiry parameter exists to remind developers to clean up / review tests that might have been forgotten. So it's probably best to set something realistic.